app/controllers/planning_applications/review/neighbour_responses_controller.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Review
+    class NeighbourResponsesController < AuthenticationController
+      include CommitMatchable
+
+      rescue_from ::Review::NotCreatableError, with: :redirect_failed_create_error
+
+      before_action :set_planning_application
+      before_action :set_consultation
+      before_action :set_neighbour_review
+
+      def show
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def edit
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def update
+        respond_to do |format|
+          format.html do
+            if @neighbour_review.update(review_params) && @consultation.update(status: consultation_status)
+              redirect_to planning_application_review_tasks_path(@planning_application), notice: t(".success")
+            else
+              render :edit
+            end
+          end
+        end
+      end
+
+      def create
+        @neighbour_review = @consultation.reviews.new(review_params.merge(specific_attributes: {consultation_type: "neighbour"}))
```

Comment:
There's a store accessor for `consultation_type` - can't you just do `{consultation_type: "neighbour"}` ?

Also, can't you use `@consultation.neighbour_reviews.new` ?